include/nfc/ndef/ndef_parser.rst

```diff
@@ -0,0 +1,51 @@
+.. _nfc_ndef_parser_readme:
+
+NFC: NDEF message and record parser
+#############################
+
+When using an NFC polling device, you can use the NDEF message parser module to interpret the NDEF messages the are read from the NFC tag. See the documentation for the :ref:`nfc_t2t_parser_readme` for more information on how to read the content of the tag.
```

Comment:
Sorry for adding it at this stage, should have been at the first review round. Anyway I would add a phrase saying that the library doesn't need to be used necessarily with an NFC poller - NDEF can also be parsed on a writable tag, to analyze the data written. Maybe it can say: "This library can also be used together with the Type 4 Tag library _(maybe insert link here)_ in a writable mode, to parse the written NDEF Message"

Reply:
added

@grochu  just to be sure, you mean a link to nfrxlib here? https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nfc/doc/api.html#nfc-tag-4-type-emulation-library

Reply:
Yes, that's right. Only this library supports Write, so we should link it here. The idea is parsing NDEF which was written by NFC Poller